### ISSUE TITLE
xloil.active_workbook() difficult to handle at startup of new Excel file

### ISSUE THREAD

**Issue by dutch-rob** [cunnane/xloil#112]
I want to check the Excel workbook file name at time of starting up before deciding to run the rest of my code.
The following:
```
self.wb = xlo.active_workbook()
if self.wb.name.startswith('AbMoP'):
    ....
```
works fine, except when you start up Excel without selecting an existing Excel file. That makes sense because there is no active workbook yet.
The xloil documentation says: "Will raise an exception if xlOil has not been loaded as an addin." (at https://xloil.readthedocs.io/en/stable/xlOil_Python/ModuleReference.html#xloil.active_workbook) 
However, when I put "self.wb = xlo.active_workbook()" in a try: block, the next code line is executed as if there was no error, and when trying to do something with self.wb, such as "if self.wb.name.startswith('AbMoP'):" the program crashes even if that is in a try: block.
It crashes to an extent that the log file in the xloil folder gets cut off mid-sentence (see attached).

I also tried:
```
wbs = xlo.workbooks
if wbs.Count > 0:
    ...
```
But then I get an error that wbs does not have a property Count.

Is there a way to catch off if Excel is starting up without active workbook?

BTW:
The xloil documentation at https://xloil.readthedocs.io/en/stable/xlOil_Python/ModuleReference.html makes a few references to Excel documentation at Microsoft but some are broken links, e.g.:
https://docs.microsoft.com/en-us/office/vba/api/excel.WorkbookWorkbook
should be
https://docs.microsoft.com/en-us/office/vba/api/excel.Workbook
similar at Excel.Workbooks, Excel.Worksheet, Excel.Window, and Excel.Windows

Best regards,
Rob

[xlOil.log2steven.txt](https://github.com/user-attachments/files/16769875/xlOil.log2steven.txt)

**Comment by cunnane** [comment]
I'll fix this but in the meantime you can workaround it with:
```
try:
   self.wb.to_com()
   if self.wb.name.startswith('AbMoP'):
     ...
except pywintypes.com_error:  # or just catch any exception
  ...
```

**Comment by dutch-rob** [comment]
The workaround works well.
Thanks very much!!
Rob

**Comment by cunnane** [comment]
This should be fixed in 0.19.1, which I just released.  `active_workbook` should now return None

**Comment by dutch-rob** [comment]
Great, thanks!!
Rob
